scripts/include-sf.js

```diff
@@ -4,7 +4,11 @@ const path = require('path');
 const shelljs = require('shelljs');
 const fs = require('fs');
 
-shelljs.exec('npm install @salesforce/cli@^1 -g');
+// Get the latest version that starts with 1 (we do not want to ever bundle version 2 of @salesforce/cli)
+// We cannot use @salesforce/cli@^1 because that will not get versions higher than what is tagged as "latest"
+const versions = JSON.parse(shelljs.exec('npm view @salesforce/cli versions --json').stdout.trim());
+const latestVersion = versions.reverse().find((v) => v.startsWith('1.'));
```

Comment:
it might get kinda weird in a `1.x.x-foo.1` scenario.  I'm ok leaving it and hoping we don't have that again.

Reply:
I think that'd be pretty unlikely but you're right that someone _could_ create a prerelease from `nightly` and that would be last. I'll add a check 👍

Reply:
Added, here is "QA"
<img width="336" alt="Screenshot 2023-04-21 at 4 13 26 PM" src="https://user-images.githubusercontent.com/1715111/233735946-4eabc8e0-1ae3-493c-8d70-85c4b15ce997.png">